# -*- coding: utf-8 -*-
# Generated by Django 1.11.29 on 2021-05-16 12:58
from __future__ import unicode_literals

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('server', '0013_plotter_harvester'),
    ]

    operations = [
        migrations.CreateModel(
            name='PlotTransfer',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('plotter_server', models.CharField(default=b'plotter-001', max_length=50, verbose_name=b'Plotter Server')),
                ('plotter_ip', models.CharField(blank=True, default=b'', max_length=20, verbose_name=b'Plotter IP')),
                ('harvester_server', models.CharField(default=b'harvester-001', max_length=50, verbose_name=b'Harvester Server')),
                ('internal_ip', models.CharField(blank=True, default=b'', max_length=20, verbose_name=b'Harvester IP')),
                ('txn_start_time', models.DateTimeField(blank=True, default=datetime.datetime.now, verbose_name=b'Start Time')),
                ('txn_stop_time', models.DateTimeField(blank=True, default=datetime.datetime.now, verbose_name=b'Start Time')),
                ('plot_check', models.BooleanField(default=False, verbose_name=b'Plot Check')),
                ('plot_check_fail_reason', models.BooleanField(default=False, verbose_name=b'Plot Check Fail Reason')),
            ],
        ),
    ]
